gym_xiangqi/xiangqi_game.py

```diff
@@ -16,16 +17,16 @@ class XiangQiGame:
     def __init__(self):
         # PyGame components
         self.running = True
-        self.FPS = 30   # for frame control in loop
-        self.winWidth = 521
+        self.FPS = 20   # loop fps
+        self.winWidth = 800
```

Comment:
Oh why not make win_width and height a constant also?

Reply:
fixed! plus, I moved FPS and the COUNT for timer to constants as well